SU-SWS/stanford_paragraph_types
# Updates the templates so they don't wrap contents in field wrappers.

# READY FOR REVIEW

# Summary
- Fixes front end display of the FWB by removing the field wrappers.

# Needed By (Date)
- July release

# Urgency
- Normal

# Steps to Test

1. Check out this branch
2. Run `drush features-import stanford_paragraph_fw_banner -y` & `drush features-import stanford_paragraph_highlight_card -y`
3. Create a FWB and review display

# Associated Issues and/or People
- EARTH-16

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
